runner/builder/builder_lcd.go

```diff
@@ -0,0 +1,87 @@
+package builder
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/mesg-foundation/engine/container"
+	"github.com/mesg-foundation/engine/ext/xos"
+	"github.com/mesg-foundation/engine/hash"
+	"github.com/mesg-foundation/engine/instance"
+	"github.com/mesg-foundation/engine/runner"
+	"github.com/mesg-foundation/engine/service"
+	runnermodule "github.com/mesg-foundation/engine/x/runner"
+)
+
+// LCD is the runner builder returning LCD msg.
+type LCD struct {
+	container    container.Container
+	engineName   string
+	port         string
+	ipfsEndpoint string
+}
+
+// NewLCD returns the runner sdk.
+func NewLCD(container container.Container, engineName, port, ipfsEndpoint string) *LCD {
+	return &LCD{
+		container:    container,
+		engineName:   engineName,
+		port:         port,
+		ipfsEndpoint: ipfsEndpoint,
+	}
+}
+
+// Start start a new runner.
+func (b *LCD) Start(owner sdk.AccAddress, srv *service.Service, env []string) (runnermodule.MsgCreate, error) {
```

Comment:
This builder package is not related to the LCD so it shouldn't return data for the LCD

Reply:
with the other modification done, I will remove it

Reply:
done